ReText/window.py

```diff
@@ -519,6 +520,107 @@ def dropEvent(e):
         self.tabWidget.dropEvent = dropEvent
         self.tabWidget.setTabBarAutoHide(globalSettings.tabBarAutoHide)
 
+        self._init_tabs_context_menu()
+
+    def _init_tabs_context_menu(self):
+        tabBar = self.tabWidget.tabBar()
+        tabBar.setContextMenuPolicy(Qt.ContextMenuPolicy.CustomContextMenu)
+        tabBar.customContextMenuRequested.connect(self._on_tab_context_menu_requested)
+
+        self._tabs_ctx_menu = QMenu(self)
+        self._tabs_ctx_menu_actions = {}
+
+        for action_type in TabActionTypes:
+            if action_type is not TabActionTypes.Unknown:
```

Comment:
Is there any reason to compare enum values with `is` / `is not` instead of `==` / `!=`?